src/app/search/state/search.service.ts

```diff
@@ -75,6 +76,38 @@ export class SearchService {
     private imageProviderService: ImageProviderService
   ) {}
 
+  /**
+   * Return -1 if a sorts before b, 1 if b sorts before a, and 0 if they are the same,
+   * comparing based on the given attribute and direction
+   * @param a: DockstoreTool or Workflow
+   * @param b: DockstoreTool or Workflow
+   * @param attribute: workflow or tool property to sort by
+   * @param direction: 'asc' or 'desc'
+   */
+  static compareAttributes(a: DockstoreTool | Workflow, b: DockstoreTool | Workflow, attribute: string, direction: string) {
```

Comment:
I don't think this should be static. We don't have static methods in other services that I can find.

Reply:
Okay. IntelliJ highlights it saying it should be static, I don't see a way to make it use anything specific to the instance, but its just a warning not an error

Reply:
I think this is all there is:

1. Remove static here
2. In other files, instead of `SearchService.compareAttributes`, invoke `searchService.compareAttributes`, where `searchService` is an instance of the service.